cooking-app/src/components/Content/Content.js

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
+import { Routes, Route, Navigate } from "react-router-dom";
+import { Container } from "../styles/Global.styled";
+import { HomePage } from "./HomePage/HomePage";
+import { ListRecipes } from "./ListRecipes/ListRecipes";
+import { RecipeBreakfast } from "./ListRecipes/RecipeBreakfast";
+import { RecipeLunch } from "./ListRecipes/RecipeLunch";
+import { RecipeDinner } from "./ListRecipes/RecipeDinner";
+import { RecipeDessert } from "./ListRecipes/RecipeDessert";
+import { Login } from "./Authorization/Login";
+import { Tips } from "./Tips/Tips";
+import { Ebook } from "./Ebook/Ebook";
+import { Search } from "./Search/Search";
+import { SingleRecipe } from "./SingleRecipe/SingleRecipe";
+import Account from "./Account/Account";
+import Register from "./Authorization/Register";
+import ForgotPassword from "./Authorization/ForgotPassword";
+
+export const Content = ({ isLoggedIn }) => {
+  return (
+    <StyledContent>
+      <Container>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route
+            path="/login"
+            element={
+              !isLoggedIn ? <Login /> : <Navigate replace to="/account" />
```

Comment:
Pomyślcie nad jakimś innym rozwiązaniem. To nie jest najlepsze. Jak będziecie mieli jakiś problem możemy o tym porozmawiać.

Reply:
Stworzyłem dwa komponenty spełniające tą samą funkcjonalność - <Redirect> i <PorotectedRoute>